.github/workflows/pr-validation.yml

```diff
@@ -0,0 +1,27 @@
+name: Pull Request Validation Workflow
+on: [pull_request]
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
```

Comment:
Maybe we can read node version dynamically like here?
https://github.com/LiskHQ/lisk-service/blob/feature/1157-prepare-lisk-service-v070-for-beta/.github/workflows/pr-coverage.yml#L12